integrated_channels/sap_success_factors/utils.py

```diff
@@ -93,3 +239,27 @@ def current_time_is_in_interval(start, end):
     interval_start = parse_lms_api_datetime(start or UNIX_MIN_DATE_STRING)
     interval_end = parse_lms_api_datetime(end or UNIX_MAX_DATE_STRING)
     return interval_start <= timezone.now() <= interval_end
+
+
+def parse_language_code(code):
```

Comment:
"Parse" doesn't really explain what is going on inside this function -- if we're doing a transform, then let's be clear about it -- something like `transform_language_code` or `transform_language_code_to_name` would work.